components/spotify/new-playlists.js

```diff
@@ -0,0 +1,50 @@
+const spotify = require("https://github.com/PipedreamHQ/pipedream/components/spotify/spotify.app.js");
+
+module.exports = {
+  name: "Spotify - New Playlists",
```

Comment:
I forgot to mention this in the other PR, but you can leave out the name of the app in the name of the source, so this can just be "New Playlists".

(I updated the YouTube sources yesterday)

Reply:
Thanks. I've updated the names.